ruler-frontend/src/main/resources/style.css

```diff
@@ -22,6 +22,10 @@ body {
     margin-left: unset !important;
 }
 
+.accordion-button.disabled {
+    pointer-events: none;
+}
```

Comment:
this was the only css solution I could come up with. My first attemt was to set disabled attribute to `button` element but the `span`s inside of it where still intercepting click events